src/utils/helpers.ts

```diff
@@ -10,6 +10,7 @@ import {
   editMessage,
 } from "../../deps.ts";
 import { Milliseconds } from "./constants/time.ts";
+import { cache } from "https://deno.land/x/discordeno@10.0.0/src/util/cache.ts";
```

Comment:
Please import from deps file
```suggestion
  editMessage,
  cache,
} from "../../deps.ts";
import { Milliseconds } from "./constants/time.ts";
```